magicbot/magicrobot.py

```diff
@@ -456,6 +456,31 @@ def _setup_vars(self, cname, component):
             setattr(component, n, injectable)
             self.logger.debug("-> %s as %s.%s", injectable, cname, n)
         
+        if hasattr(component, '__annotations__'):
+            for n, type_ in component.__annotations__.items():
+                if n.startswith('_') or isinstance(getattr(type(component), n, True), property):
```

Comment:
umm...yes? Although technically this allows you to create a variable of instance `property` without assigning it to a variable, which is what is currently allowed. So by removing this then 

```
foo = tunable # Works
bar: tunable # Does not work
```
I think...99% sure.